clang/include/clang/CIR/Passes.h

```diff
@@ -18,6 +18,11 @@
 #include <memory>
 
 namespace cir {
+/// Create a pass for transforming CIR operations to more 'scf' dialect-friendly
+/// forms. It rewrites operations that aren't supported by 'scf', such as breaks
+/// and continues.
+std::unique_ptr<mlir::Pass> createMLIRLoweringPreparePass();
```

Comment:
Let's call this pass `MLIRCoreDialectsLoweringPrepare`. We already have a poor name choice might be good to avoid making it worse (we should rename things at some point).